src/amo/pages/Home/index.js

```diff
@@ -282,7 +286,7 @@ export class HomeBase extends React.Component {
           />
         )}
 
-        {includeFeaturedThemes && (
+        {includeFeaturedThemes && showThemes && (
```

Comment:
what about the FeaturedCollection section (see ~ https://github.com/mozilla/addons-frontend/pull/6857/files#diff-9d92c2d79f19315442c24a9765c04dd7L50)? I've seen themes there too (not a lot I don't think but sometimes). In this PR you can see it was removed https://github.com/mozilla/addons-frontend/commit/c169c8a4e7d87a08f53b3e30e3b38c6afa7a406b#diff-9d92c2d79f19315442c24a9765c04dd7 (I couldn't easily get access to the one where it was added but hopefully this shows what I mean).

Maybe this is okay for now since it's not currently set up this way..? What do you think?

Reply:
I think we cannot hide what we don't know..

Reply:
okay I just didn't want anything to sneak in inadvertently later on so just double checking :)

Reply:
Collections are not something "theme" or "extension" specific unfortunately, so we cannot really do anything anyway...

Reply:
oh, interesting. Then, let me file an issue for that because this PR gets too big.

Reply:
Can we defer this change to https://github.com/mozilla/addons-frontend/issues/6870? I think it is fine given that we don't have any featured "theme" collections currently.